modules/appeals_api/spec/models/notice_of_disagreement_spec.rb

```diff
@@ -6,13 +6,13 @@
 describe AppealsApi::NoticeOfDisagreement, type: :model do
   include FixtureHelpers
 
-  let(:notice_of_disagreement) { build(:notice_of_disagreement, form_data: form_data, auth_headers: auth_headers) }
-
-  let(:auth_headers) { default_auth_headers }
-  let(:form_data) { default_form_data }
+  let(:notice_of_disagreement) do
+    review_option = form_data['data']['attributes']['boardReviewOption']
```

Comment:
`form_data` comes from a `let` below this one. Let's move `let(:form_data)` up to above this line so we can follow where it came from more easily.